indico/modules/events/registration/models/registrations.py

```diff
@@ -445,13 +445,6 @@ def can_be_withdrawn(self):
     def data_by_field(self):
         return {x.field_data.field_id: x for x in self.data}
 
-    @property
-    def picture_data(self):
-        """Return the picture data in personal data."""
-        rdata = next((d for d in self.data if d.field_data.field.personal_data_type == PersonalDataType.picture), None)
-        if rdata and rdata.storage_file_id is not None:
-            return rdata
```

Comment:
Can you copy this logic+docstring into the other method? It's pretty much the same but I think it's nicer than the other ;)

Reply:
Done.